### DISCUSSION TITLE
6주차 회고

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#224] [회고]
### TRY(PROBLEM의 해결책으로 다음 회고까지 실행할 점)

아래 양식을 복사해 댓글로 작성해주시면 됩니다.
```
### KEEP

### PROBLEM

### TRY
```

**Comment by devwqc** [comment]
### KEEP
- 프로젝트 마무리가 되었는데 다들 너무 친절하다. 사실 마무리 다 돼서 프로젝트 완성도가 부족하면 서로 탓하고 그럴 수 있는데 우리 팀원들은 모두 천성이 친절한 거 같다.

### PROBLEM
- 시간이 조금 더 있었으면 하는 아쉬움이 있지만 시간이 더 있었어도 같은 생각을 했을 거 같다.

### TRY
- 지금 우리가 할 수 있는 최선을 다하자! 파이팅👍

**Comment by Seoin02** [comment]
### KEEP
- 프로젝트를 잘 마무리하기 위해 새벽까지 함께 코딩하는 게 즐거웠다.
- 마지막까지 본인이 맡은 역할과 페이지 구현에 최선을 다하는 모습이 멋지다.

### PROBLEM
- QA 시 이슈가 많이 발생했다.

### TRY
- 얼른 이슈를 해결해야겠다.
